src/common/lib/polkadot-provider/query/genetic-analysis.js

```diff
@@ -0,0 +1,14 @@
+export async function geneticAnalysisByOwner(api, accountId) {
+  let res = (await api.query.geneticAnalysis.geneticAnalysisByOwner(accountId)).toHuman()
+  return res
+}
+
+export async function geneticAnalysisStorage(api, trackingId) {
+  let res = (await api.query.geneticAnalysis.geneticAnalysisStorage(trackingId)).toHuman()
```

Comment:
why are you using `let` instead of const and `(`? please update with better practice